tests/unit/test_zfsacl.py

```diff
@@ -0,0 +1,113 @@
+import os
+import pwd
+import pytest
+import shutil
+import truenas_pylibzfs
+import zfsacl
+from contextlib import contextmanager
+from truenas_api_client import Client, ClientException
+
+
+@pytest.fixture(scope='module')
+def local_user():
+    with Client() as c:
+        usr = c.call('user.create', {
+            'username': 'test_local_user',
+            'full_name': 'test_local_user',
+            'random_password': True,
+            'group_create': True
+        })
+        try:
+            yield usr
+
+        finally:
+            c.call('user.delete', usr['id'])
+
+
+@pytest.fixture(scope='module')
+def create_dataset():
+    """ create an nfsv4 dataset under /root """
```

Comment:
But it's not under `/root` is it?  AFAICS it's under var in the boot pool, mounted as `/var`.